composer.json

```diff
@@ -21,8 +21,8 @@
         "php": "^5.6 || ^7.0",
         "symfony/console": "^3.2.7",
         "zendframework/zend-code": "^3.1 || ^2.6.3",
-        "zendframework/zend-component-installer": "^0.7.1",
-        "zendframework/zend-expressive": "^2.0 || dev-2.0.X@dev",
+        "zendframework/zend-component-installer": "^1.0",
```

Comment:
I'd argue this should be `"^1.0 || ^0.7.1"`, which will allow for previous versions if they are already installed.